scripts/ci-e2e.sh

```diff
@@ -86,22 +87,21 @@ create_powervs_network_instance(){
     ibmcloud pi service-target ${CRN}
 
     # Create the network instance
-    ibmcloud pi network-create-public ${IBMPOWERVS_NETWORK_NAME} --dns-servers "8.8.8.8 9.9.9.9"
+    capibmadm powervs network create --name ${IBMPOWERVS_NETWORK_NAME} --service-instance-id ${IBMPOWERVS_SERVICE_INSTANCE_ID} --zone ${ZONE}
```

Comment:
@Karthik-K-N @Amulyam24 don't we support dns option to the network create? if not lets create an issue to enhance that.

Reply:
There is an option to pass dns-servers - https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/6b35dda78e74c17a53876c4d6428c9f2c768d1eb/cmd/capibmadm/cmd/powervs/network/create.go#L67